mobile/mobile_40.asm

```diff
@@ -1530,7 +1530,11 @@ Function1009f3:
 _LinkBattleSendReceiveAction:
 	call .StageForSend
 	ld [wLinkBattleSentAction], a
-	farcall PlaceWaitingText
+;	farcall PlaceWaitingText
+	ld a, BANK(PlaceWaitingText)
+	ld hl, PlaceWaitingText
+	vc_hook send_byt2
+	rst FarCall
```

Comment:
I remember doing this but I'm still not sure if this is the best way to declar this hook. Maybe keeping the farcall (i.e. putting the `vc_hook` macro right before it) and modifying the patch template to add an offset of +5 to the address is a better idea.

Reply:
https://github.com/pret/pokecrystal/pull/882/commits/dc2001290adbbd930358055409697cf75b4c36e9 should resolve this.